### PR TITLE
Added prequel quotes & trilogy keys

### PR DESCRIPTION
**Pull request by Omnigazer**

### CODE REVIEW

File: starWarsQuotesRepository.py [hunk 1]
```diff
@@ -20,19 +20,24 @@ def __init__(
...
+    def __getQuotes(self, trilogy: str=None) -> List:
         jsonContents = self.__readJson()
 
         quotes = jsonContents.get("quotes")
         if not utils.hasItems(quotes):
             raise ValueError(f'JSON contents of quotes file \"{self.__quotesFile}\" is missing \"quotes\" key')
 
-        return quotes
+        if trilogy not in quotes.keys():
+            result = [item for sublist in quotes.values() for item in sublist]
```
Comment (charlesmadere) [inline starWarsQuotesRepository.py:36]: my python sux so this is hard to read, but hey, i think i get what's going on here, looks dope 👍

File: starWarsQuotesRepository.py [hunk 2]
```diff
@@ -20,19 +20,24 @@ def __init__(
 
         self.__quotesFile = quotesFile
 
-    def fetchRandomQuote(self) -> str:
-        jsonContents = self.__getQuotes()
+    def fetchRandomQuote(self, trilogy: str=None) -> str:
```
Comment (charlesmadere) [inline starWarsQuotesRepository.py]: tiny nitpick: can you add spaces before and after the `=`?

File: starWarsQuotesRepository.py [hunk 3]
```diff
@@ -20,19 +20,24 @@ def __init__(
...
+        jsonContents = self.__getQuotes(trilogy)
         randomIndex = random.randint(0, len(jsonContents) - 1)
         return jsonContents[randomIndex]
 
-    def __getQuotes(self) -> List:
+    def __getQuotes(self, trilogy: str=None) -> List:
```
Comment (charlesmadere) [inline starWarsQuotesRepository.py]: tiny nitpick again, please add spaces before and after the `=`

**Review by charlesmadere — CHANGES_REQUESTED** [review]
just two super tiny things!! thanks for the code 🤓

File: starWarsQuotesRepository.py (hunk 1 above)
Comment (charlesmadere) [reply starWarsQuotesRepository.py:36]: (no changes needed)

Comment (Omnigazer) [reply starWarsQuotesRepository.py:36]: This in particular is just a "flatten()" function i googled for python, maybe there's a better / more concise way.

**Review by charlesmadere — APPROVED** [review]

**Pull request merged**